File: Offer/7.py
class Solution:
    def Fibonacci(self, n):
        if n < 2:
            return n
        f1 = 0
        f2 = 1
        for _ in range(2, n + 1):
            f3 = f1 + f2
            f1 = f2
            f2 = f3
        return f3

File: Offer/CPP/7.cpp
class Solution
{
  public:
    int Fibonacci(int n)
    {
        if (n < 2)
            return n;
        int a = 0, b = 1, result = 0;
        for (int i = 1; i < n; i++)
        {
            result = a + b;
            a = b;
            b = result;
        }
        return result;
    }
};
